doc/api/toolkits/mplot3d/view_angles.rst

```diff
@@ -38,3 +38,119 @@ further documented in the `.mplot3d.axes3d.Axes3D.view_init` API.
+corresponded directly to azimuth and elevation; this is also how it is done
+in `MATLAB <https://www.mathworks.com/help/matlab/ref/view.html>`_.
+This approach works fine for polar plots, where the *z* axis is special;
+however, it leads to a kind of 'gimbal lock' when looking down the *z* axis:
+the plot reacts differently to mouse movement, dependent on the particular
+orientation at hand. Also, 'roll' cannot be controlled.
+
+As an alternative, there are various mouse rotation styles where the mouse
+manipulates a 'trackball'. In its simplest form (``mouserotationstyle: trackball``),
+the trackball rotates around an in-plane axis perpendicular to the mouse motion
+(it is as if there is a plate laying on the trackball; the plate itself is fixed
+in orientation, but you can drag the plate with the mouse, thus rotating the ball).
+This is more natural to work with than the ``azel`` style; however,
+the plot cannot be easily rotated around the viewing direction - one has to
+drag the mouse in circles with a handedness opposite to the desired rotation.
+
+A different variety of trackball rotates along the shortest arc on the virtual
+sphere (``mouserotationstyle: arcball``); it is a variation on Ken Shoemake's
+ARCBALL [Shoemake1992]_. Rotating around the viewing direction is straightforward
+with it. Shoemake's original arcball is also available
+(``mouserotationstyle: Shoemake``); it is free of hysteresis, i.e.,
+returning mouse to the original position returns the figure to its original
+orientation, the rotation is independent of the details of the path the mouse
+took. However, Shoemake's arcball rotates at twice the angular rate of the
+mouse movement (it is quite noticeable, especially when adjusting roll).
+So it is a trade-off.
+
+Shoemake's arcball has an abrupt edge; this is remedied in Holroyd's arcball
+(``mouserotationstyle: Holroyd``).
+
+Henriksen et al. [Henriksen2002]_ provide an overview.
+
+In summary:
+
+.. list-table::
+   :width: 100%
+   :widths: 30 20 20 20 35
+
+   * - Style
+     - traditional [1]_
+     - incl. roll [2]_
+     - uniform [3]_
+     - path independent [4]_
+   * - azel
+     - ✔️
+     - ❌
+     - ❌
+     - ✔️
+   * - trackball
+     - ❌
+     - ～
+     - ✔️
+     - ❌
+   * - arcball
+     - ❌
+     - ✔️
+     - ✔️
+     - ❌
+   * - Shoemake
+     - ❌
+     - ✔️
+     - ✔️
+     - ✔️
+   * - Holroyd
+     - ❌
+     - ✔️
+     - ✔️
+     - ✔️
+
+
+.. [1] The way it was historically; this is also MATLAB's style
+.. [2] Mouse controls roll too (not only azimuth and elevation)
+.. [3] Figure reacts the same way to mouse movements, regardless of orientation (no difference between 'poles' and 'equator')
+.. [4] Returning mouse to original position returns figure to original orientation (no hysteresis: rotation is independent of the details of the path the mouse took)
+
+Try it out by adding a file ``matplotlibrc`` to folder ``matplotlib\galleries\examples\mplot3d``,
```

Comment:
Could you replace this section with a short self-contained example someone could paste into a script and run?

Reply:
I'm thinking of something that has something like the following at the top of the snippet, since this specific rcparam isn't demonstrated anywhere yet:

```
import matplotlib as mpl
mpl.rcParams['axes3d.mouserotationstyle'] = 'azel'
```

Reply:
Ah, yes, of course! Excellent idea.